app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -653,14 +653,14 @@ private String getIVText() {
                 returnVal += "<br>"; //breakline
                 //for following stage evolution (example, dratini - dragonair - dragonite)
                 //if the current evolution has another evolution calculate its range and break
-                for (Pokemon nextEvo : evolution.evolutions) {
+                if(evolution.evolutions.size()>0) {
```

Comment:
Coding style is kept by Android Studio formatter. But in this case I didn't want to auto format because it would reformat the whole file and cause merging conflicts. So it can be done later and also it isn't even worth mentioning.